src/pages/WorldEditor/WorldEditor.tsx

```diff
@@ -40,7 +40,7 @@ export const WorldEditor: React.FC = () => {
     <div className="WorldEditor">
       <WithNavigationBar hasBackButton withSchedule>
         <AdminRestricted>
-          <WorldNav />
+          {worldId && <WorldNav />}
```

Comment:
Hiding and showing the navigation creates more UI issues than simply disabling the buttons. Both approaches, however, do not solve the issue of someone simply typing the URL in the address bar.

Reply:
That's true we need to think about that as well. Keep in mind that most of our buttons use history.goBack and opening a url directly would break those buttons and make them do nothing.

Reply:
> most of our buttons use history.goBack and opening a url directly would break those buttons and make them do nothing.

Looks like an opportunity for future PR to make the navigation more robust.